<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>1082</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string> 로그인 및 계좌관리 </string>
    </property>
    <widget class="QPushButton" name="loginBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌비번</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginStatus">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>18</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>로그인
 실패</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>631</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>실시간 감시 상황판</string>
    </property>
    <widget class="QTableWidget" name="watchOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>611</width>
       <height>251</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>0</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>종목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>틱 이평비교</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>지수 이평비교</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>체결강도 비교</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>진입 가능여부</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>진입 상태</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> 감시할 종목코드</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="codeInput">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="watchBtn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>21</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>감시 시작</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopBtn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>21</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>중지</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>1. 200틱 &gt; 400틱 이평일때 
2. 코스피 2분 &gt; 4분 이평일때
3. 3분 체결강도 &gt; 70 일때</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="autoCheck">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>42</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>자동진입 여부</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>631</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>비교 로그</string>
    </property>
    <widget class="QPlainTextEdit" name="compareLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>611</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>631</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>거래대금 상위 종목</string>
    </property>
    <widget class="QPlainTextEdit" name="tmoneyOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>611</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="upjongMenu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>전체</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>코스피</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>코스닥</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="gwanjongMenu">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>관리종목 포함</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>관리종목 미포함</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="tmoneyBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>291</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>수동주문</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>주문</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종목</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종류</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>수량</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>가격</string>
     </property>
    </widget>
    <widget class="QComboBox" name="accList">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="orderType">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>신규매수</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>신규매도</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매수취소</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매도취소</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="orderCode">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="hoga">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>시장가</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>지정가</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="orderNum">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="orderPrice">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000000</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="orderBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>현금주문</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>291</width>
      <height>601</height>
     </rect>
    </property>
    <property name="title">
     <string>실시간 호가 및 로그</string>
    </property>
    <widget class="QTableWidget" name="bidaskTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>271</width>
       <height>371</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>매도 5호가</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>매도 4호가</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>매도 3호가</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>매도 2호가</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>매도 1호가</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>현재가</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>매수 1호가</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>매수 2호가</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>매수 3호가</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>매수 4호가</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>매수 5호가</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>매도</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매수</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>매수/매도 로그</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="bidaskLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>271</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="rHogaSearch">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rHogaCode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="rstopBtn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>중지</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
